crates/connectors/ndc-postgres/src/configuration/version2.sql

```diff
@@ -109,12 +109,14 @@ WITH
            WHEN att.attidentity = 'a' THEN 'identityAlways'
            ELSE 'notIdentity'
       END
-      AS is_identity
-      -- skipped because yugabyte is based on pg11 that does not have this field.
-      -- CASE WHEN att.attgenerated = 's' THEN 'isGenerated' ELSE 'notGenerated' END
-      -- AS is_generated
+      AS is_identity,
+      CASE WHEN attgenerated_exists
+	       THEN CASE WHEN attgenerated::text = 's' THEN 'isGenerated' ELSE 'notGenerated' END
+           ELSE 'notGenerated'
+      END as is_generated
```

Comment:
Seems like a strange way to implement booleans. Why not `true` and `false`?

Reply:
Just liked being explicit, wouldn't mind changing it to bool if you want me to.

Reply:
I've changed this so instead of returning `isGenerated`, it will return `stored`, so that if and when postgres adds additional values to this, it will be strictly additive.